pandas/tests/computation/test_eval.py

```diff
@@ -1345,6 +1345,38 @@ def test_multi_line_expression_local_variable(self):
         assert_frame_equal(expected, df)
         assert ans is None
 
+    def test_multi_line_expression_callable_local_variable(self):
+        df = pd.DataFrame({'a': [1, 2, 3], 'b': [4, 5, 6]})
+
+        def local_func(a, b):
+            return b
+
+        expected = df.copy()
+        expected['c'] = expected['a'] * local_func(1, 7)
+        expected['d'] = expected['c'] + local_func(1, 7)
+        ans = df.eval("""
+        c = a * @local_func(1, 7)
+        d = c + @local_func(1, 7)
+        """, inplace=True)
+        assert_frame_equal(expected, df)
+        assert ans is None
+
+    def test_multi_line_expression_callable_local_variable_with_kwargs(self):
+        df = pd.DataFrame({'a': [1, 2, 3], 'b': [4, 5, 6]})
```

Comment:
add the issue reference (is there an issue open for this? if not add the pr number) as a comment

Reply:
I'm not sure what I'm supposed to add here. Something like ``# Github PR #26426 `` ?

I couldn't find a related issue.

Reply:
```
# 26526
```

is enough